tests/probes/yamlfilecontent/test_probes_yamlfilecontent.sh

```diff
@@ -0,0 +1,33 @@
+#!/usr/bin/env bash
+
+. $builddir/tests/test_common.sh
+
+function test_probes_yamlfilecontent {
+
+    probecheck "yamlfilecontent" || return 255
```

Comment:
This line causes the test to be skipped.

Removing it will run the test, and cause fail due to invalid schema:
```
File '/home/wsato/git/openscap/tests/probes/yamlfilecontent/test_probes_yamlfilecontent.xml' line 40: Element '{http://oval.mitre.org/XMLSchema/oval-definitions-5#independent}yamlfilecontent_test': This element 
is not expected. Expected is one of ( {http://oval.mitre.org/XMLSchema/oval-definitions-5}test, {http://oval.mitre.org/XMLSchema/oval-definitions-5#aix}interim_fix_test, {http://oval.mitre.org/XMLSchema/oval-def
initions-5#aix}fileset_test, {http://oval.mitre.org/XMLSchema/oval-definitions-5#aix}fix_test, {http://oval.mitre.org/XMLSchema/oval-definitions-5#aix}no_test, {http://oval.mitre.org/XMLSchema/oval-definitions-5
#aix}oslevel_test, {http://oval.mitre.org/XMLSchema/oval-definitions-5#apache}httpd_test, {http://oval.mitre.org/XMLSchema/oval-definitions-5#catos}line_test, {http://oval.mitre.org/XMLSchema/oval-definitions-5#
catos}module_test, {http://oval.mitre.org/XMLSchema/oval-definitions-5#catos}version55_test ).
```

Reply:
When we will have the probe test will start to fail and we will be fixing it, that line is OK. But, what we could do to fix the schema *before* we officially commit and publish it in OVAL?

Reply:
I think we can add OVAL 5.11.3 schemas (as a future version) and then modify them. We should add proper comments there.

Reply:
@evgenyz @yuumasato We need an agreement on the schemas, because it's a blocker for both test and implementation (you can temporarily do --skip-valid but I don't want to see anything like that in our repo). Do you agree with my proposal from the previous comment? If yes, I can create a separate PR for schemas.

Reply:
@jan-cerny Yes, your suggestion makes sense to me.

Reply:
See https://github.com/OpenSCAP/openscap/pull/1482